docs/UserGuide.md

```diff
@@ -95,39 +95,82 @@ Format: `unmark INDEX`
 
 Example: `list` followed by `unmark 3` marks the 3rd task in the task list as incomplete.
 
-### Editing a task : `edit`
+### 3.8 Editing a task : `edit`
 
-Edits the specified task in the task list.
+Edits an existing task in the task list.
 
-Format: `edit INDEX [d/DESCRIPTION] [t/TAG]`
-- Edits the task at the specified `INDEX`. The index refers to the index number shown in the displayed task list. The index **must be a positive integer** 1, 2, 3, …​
-- At least one of the optional fields must be provided.
-- Existing values will be updated to the input values.
-- When editing tags, the existing tags of the task will be removed i.e adding of tags is not cumulative.
-- You can remove all the task’s tags by typing `t/` without specifying any tags after it.
+Format: `edit INDEX [n/NAME] [d/DESCRIPTION] [dl/DEADLINE] [p/PRIORITY] [t/TAG]...`
+- `DEADLINE` should be in the format `YYYY-MM-DD`.
+- `PRIORITY` should be one of `low`, `medium`, `high`.
+- `TAG` should not contain any space.
 
-Example: `edit 2 d/Prepare for CS2103T Tutorial` Edits the description of the 2nd task to be `Prepare for CS2103T Tutorial`
+Examples:
+- `edit 1 d/Prepare for CS2103T tutorial`
+- `edit 2 n/CS2103T meeting t/CS2103T`
```

Comment:
Should we use 'Example' instead of 'Examples' for consistency?

Reply:
Sounds good! Will change soon.